core/src/calleeFunctions/WstETHCurveUniv3Callee.ts

```diff
@@ -31,39 +26,22 @@ const getCalleeData = async function (
 const getMarketPrice = async function (
     network: string,
     collateral: CollateralConfig,
-    amount: BigNumber
+    collateralAmount: BigNumber
 ): Promise<BigNumber> {
     // convert wstETH into stETH
```

Comment:
For all the other conversions we have separate functions to reuse them. Would it make sense to introduce one here too or will we probably not have to reuse this conversion?

Reply:
We can refactor it as soon as we need to reuse it. Current assumption: conversions happening here wouldn't be reusable outside of this function